src/redux/selectors.js

```diff
@@ -1,3 +1,23 @@
+import { isWithinRange } from 'date-fns';
+import { find } from 'lodash';
+
+const getVelocity = (stories, sprint) =>
+  stories.reduce((result, story) => {
+    if (isWithinRange(story.closedAt, sprint.startsAt, sprint.endsAt)) {
+      return result + story.estimate;
+    }
+
+    return result;
```

Comment:
🎨: Maybe an implicit return arrow in the `reduce` for `isWithinRange() ? result + story.estimate : result`?